crates/wasi-common/src/virtfs.rs

```diff
@@ -256,7 +256,7 @@ impl VecFileContents {
 pub struct InMemoryFile {
     cursor: RefCell<types::Filesize>,
     parent: Rc<RefCell<Option<Box<dyn VirtualFile>>>>,
-    fd_flags: types::Fdflags,
+    fd_flags: RefCell<types::Fdflags>,
```

Comment:
This is a `Copy` type I think, so could this perhaps be a `Cell<types::Fdflags>` to entirely elimitate the possibility of panics?

Reply:
Yep, thanks!